### PR TITLE
42 linting

### PR DESCRIPTION
**Pull request by kapaha**
I've add ESLint and Prettier to keep our code style and formatting consistent.

### CODE REVIEW

**Review by Ocoldwell — APPROVED** [review]
Looking good to me.

**Review by dwarjie — APPROVED** [review]
Yep! Working properly. Just forgot the npm install.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# OS generated files #
+######################
+.DS_Store
+.DS_Store?
+
+# Dependency directories #
+##########################
+node_modules/
```
Comment (tanselbay1) [inline .gitignore:8]: test

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
```
Comment (tanselbay1) [inline .eslintrc.js:1]: review testing

**Pull request merged**